### PR TITLE
chore(moderation): Adjust footer length with datatable width (columns)

### PR DESCRIPTION
**Pull request by maierthomas**
fix footer length (width) in moderation view

closes eclipse/sw360#297

### CODE REVIEW

File: frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp [hunk 1]
```diff
@@ -57,35 +57,37 @@
                 <div id="tab-Open" class="tab-pane">
                     <table id="moderationsTable" cellpadding="0" cellspacing="0" border="0" class="display" style="width:100%">
                         <colgroup>
-                            <col style="width: 5%;" />
+                            <col style="width: 90px;" />
```
Comment (maxhbr) [inline frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp]: mixing `px` and `%` is normally not a good idea

File: frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp [hunk 2]
```diff
@@ -209,6 +211,12 @@
                 {title: "Moderators", render: {display: renderModeratorsListExpandable}},
                 {title: "State"},
                 {title: "Actions"}
+            ],
+            columnDefs: [
+                {
+                    width: "90px", "targets": [0],
```
Comment (maxhbr) [inline frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp]: Why does datables needs the width here? does it ignoire the width stated in the html above?

File: frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp (hunk 1 above)
Comment (maxhbr) [reply frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp]: How does it behave when the window is scaled?

Comment (maierthomas) [reply frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp]: yes mixing of px and % is a bad idea.
Anyway I don't like to specify each column width.
I removed this code, thus the browser calculates the optimal column width.
I looks much better, especially the date and type columns are now calculates in one line.

File: frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp (hunk 2 above)
Comment (maierthomas) [reply frontend/sw360-portlet/src/main/webapp/html/moderation/view.jsp]: just a test, removed code.

**Comment by adrian-evo** [comment]
Tested. Footer adjusted successfully.

**Pull request merged**